test/libsolidity/Metadata.cpp

```diff
@@ -43,7 +43,9 @@ BOOST_AUTO_TEST_CASE(metadata_stamp)
 		}
 	)";
 	CompilerStack compilerStack;
-	BOOST_REQUIRE(compilerStack.compile(std::string(sourceCode)));
+	compilerStack.addSource("", std::string(sourceCode));
+	/// NOTE: compiles without optimisations
```

Comment:
Yep, will make another PR for this one and please make a PR for the GasMeter if optimisations should be enabled for it (since you have worked with that code before).